src/css/controls/imports/controlbar.less

```diff
@@ -82,13 +82,23 @@
 }
 
 .jw-button-container {
+    display: block;
```

Comment:
Would be better to use autoprerixer to compile backwards compatible support:

From https://autoprefixer.github.io/
```css
    display: -webkit-box;
    display: flex;
    -webkit-box-orient: horizontal;
    -webkit-box-direction: normal;
            flex-flow: row nowrap;
    -webkit-box-flex: 1;
            flex: 1 1 auto;
    -webkit-box-align: center;
            align-items: center;
    -webkit-box-pack: center;
            justify-content: center;
    width: 100%;
    padding: 0 12px;
```

Reply:
sounds like you'd prefer a manual addition where needed? i tend to agree, autoprefixr could really bloat